src/applications/coronavirus-screener/sass/coronavirus-screener.scss

```diff
@@ -1,11 +1,26 @@
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 
-.covid-screener-results {
-  background-color: $color-gray-dark;
-  &.covid-screener-results-pass {
-    i {font-size: 500%;}
+.covid-screener {
+  h1,h2,h3,h4,h5,li,p {
+    font-family: Source Sans Pro,Helvetica Neue,Helvetica,Roboto,Arial,sans-serif !important;
+  }
+  .covid-screener-date {
+    margin: 2em 0 2em 0;
+    h2,h3,h4 { margin: 0; }
+    h4 { font-size: 150%; }
+  }
+  .covid-screener-results {
+    h3 { font-size: 300%; }
+    background-color: $color-primary-alt-lightest;
+    p {
+      font-weight: bold !important;
```

Comment:
it would be nice to remove this `!important` but it looks necessary due to upstream issue https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/issues/274

Reply:
Yup.  Once someone adds `!important` it's all downhill from there.